### PR TITLE
docs: add instructions on upgrading to the latest release (#450)

### PR DESCRIPTION
**Pull request by SOG-web**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [ ] create-analog
- [ ] router
- [ ] platform
- [ ] content
- [ ] nx-plugin
- [ ] trpc

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/docs-app/docs/getting-started.md [hunk 1]
```diff
@@ -165,3 +165,33 @@ The server for the API/SSR build artifacts is located in the `dist/apps/analog-a
 
 </TabItem>
 </Tabs>
+
+## Upgrade your existing application
```
Comment (brandonroberts) [inline apps/docs-app/docs/getting-started.md:169]: This should be a separate page under the `Features` group.

You can create a folder for `updating` [in here](https://github.com/analogjs/analog/tree/main/apps/docs-app/docs/features) and an `overview.md` for the content.

The config file for the sidemenu is here https://github.com/analogjs/analog/blob/main/apps/docs-app/sidebars.js

Comment (brandonroberts) [inline apps/docs-app/docs/getting-started.md:169]:
```suggestion
## Updating to the latest version
```

File: apps/docs-app/docs/getting-started.md [hunk 2]
```diff
@@ -165,3 +165,33 @@ The server for the API/SSR build artifacts is located in the `dist/apps/analog-a
...
+
+There are two methods for upgrading an Analog application. You have the option to utilize the `ng update @analogjs/platform` command
+to Upgrade a standalone project, or you can make use of the Nx migrate command, which is `nx migrate @analogjs/platform`.
```
Comment (brandonroberts) [inline apps/docs-app/docs/getting-started.md:172]:
```suggestion
You can use the `ng update` command for an Angular CLI workspace, or the `nx migrate` command for updating within an Nx workspace.
```

File: apps/docs-app/docs/getting-started.md [hunk 3]
```diff
@@ -165,3 +165,33 @@ The server for the API/SSR build artifacts is located in the `dist/apps/analog-a
...
+
+<Tabs groupId="app-upgrader">
+  <TabItem label="ng update" value="ng-update">
+
+### ng update
+
+Upgrade an Analog project with the following `ng update` command:
```
Comment (brandonroberts) [inline apps/docs-app/docs/getting-started.md:179]:
```suggestion
To update an Analog project with the `ng update` command:
```

File: apps/docs-app/docs/getting-started.md [hunk 4]
```diff
@@ -165,3 +165,33 @@ The server for the API/SSR build artifacts is located in the `dist/apps/analog-a
...
+```shell
+ng update @analogjs/platform
+```
+</TabItem>
+
+  <TabItem label="Nx" value="nx-migrate">
+
+### Nx Migrate
+
+Upgrade an Analog project with the following `nx migrate` command:
```
Comment (brandonroberts) [inline apps/docs-app/docs/getting-started.md:190]:
```suggestion
To update an Analog project with the `nx migrate` command: 
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: apps/docs-app/docs/getting-started.md (hunk 1 above)
Comment (SOG-web) [reply apps/docs-app/docs/getting-started.md:169]: ok, i will fix that. Thanks

**Comment by brandonroberts** [comment]
@SOG-web are you still able to move this to its own page?

**Comment by SOG-web** [comment]
Am sorry for fixing it late, being really busy lately

Am done moving it to a new page

Where on the feature route list should I include it. 

Should come first before routing or  after routing or comes last on the list

**Comment by brandonroberts** [comment]
No problem. It can be last below the `Testing` section

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks! @allcontributors add @SOG-web for doc

**Pull request merged**